pkg/controller/atlasfederatedauth/atlasfederated_auth_controller.go

```diff
@@ -50,16 +47,16 @@ type AtlasFederatedAuthReconciler struct {
 func (r *AtlasFederatedAuthReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.With("atlasfederatedauth", req.NamespacedName)
 
-	fedauth := &mdbv1.AtlasFederatedAuth{}
-	result := customresource.PrepareResource(ctx, r.Client, req, fedauth, log)
+	fedauth := mdbv1.AtlasFederatedAuth{}
```

Comment:
let's continue using a pointer (will change)

Reply:
:+1: I changed it to validate a case with the test and forgot to revert